docs/Changelog.md

```diff
@@ -31,21 +31,7 @@
 * [#6750](https://github.com/TouK/nussknacker/pull/6750) Add varargs to `#COLLECTION.concat` and `#COLLECTION.merge`.
 * [#6778](https://github.com/TouK/nussknacker/pull/6778) SpeL: check for methods if a property for a given name does not exist.
 * [#6769](https://github.com/TouK/nussknacker/pull/6769) Added possibility to choose presets and define lists for Long typed parameter inputs in fragments.
-* [#6807](https://github.com/TouK/nussknacker/pull/6807) Add conversion functions to primitives to: `#CONV`:
-  * toNumberOrNull
-  * toString
-  * toBoolean
-  * toBooleanOrNull
-  * toInteger
-  * toIntegerOrNull
-  * toLong
-  * toLongOrNull
-  * toDouble
-  * toDoubleOrNull
-  * toBigInteger
-  * toBigIntegerOrNull
-  * toBigDecimal
-  * toBigDecimalOrNull
+* [#6807](https://github.com/TouK/nussknacker/pull/6807) Add `toNumberOrNull` conversion function, to `#CONV`.
```

Comment:
Because I added missing `toNumberOrNull` to already existing `toNumber` for the sake of consistency with other conversion methods. If we want only use conversions as extension methods then we have to deprecated `toNumber` method and then I can remove `toNumberOrNull` so the questions is what should I do?